src/Core/Geometry/TopologicalMesh.cpp

```diff
@@ -162,11 +166,13 @@ void copyAttribToCore( TriangleMesh& triMesh, const HandleAndValueVector<T>& dat
 
 //! [Default command implementation]
 struct DefaultNonManifoldFaceCommand {
+    /// \brief details string is printed along with the message
+    DefaultNonManifoldFaceCommand( std::string details = {} ) : m_details {details} {}
```

Comment:
I would prefer that we pass a `struct TopologicalMesh::NonManifoldFaceCommandContext` instead of a `std::string`, which will allow us to add more attributes if needed, and without changing the API.